fixtures/beamz/ps10w-led-pin-spot-10w-4-in-1-dmx.json

```
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "PS10W LED PIN SPOT 10W 4-IN-1 DMX",
```

```suggestion
  "name": "PS10W LED Pin Spot 10W RGBW DMX",
```

Beamz seems to use a slightly different (but equally long and complex!) name on every page.  This is what's in the manual, anyway.  I could go either way on this, but whatever we pick, I think it's easier to read if "Pin Spot" isn't all-caps.